src/repositories/interfaces.ts

```diff
@@ -1,9 +1,38 @@
 import { BaseEntity } from 'typeorm';
 
+import { ResultOf } from '@graasp/sdk';
+
 export interface GetAll<T extends BaseEntity> {
   /**
    * Retrieves all the entities.
    * @returns An array containing all the entities.
    */
   getAll(): Promise<T[]>;
 }
+
+export interface GetForItem<T extends BaseEntity> {
```

Comment:
I'm nitpicking here so don't hesitate to ignore if you think it is not worthy.

I feel like `getForItem(...)` is a confusing name to me. I've the habit of having `getByItem(...)` or `getByItemId(...)`.

When I see `getForItem` I'm never totally sure of the use of this function. "Does it get something in a format that satisfy an Item, or does it get something related to an Item ?"

Just to make a comparaison (I didn't made a profesionnal analysis) from the search bar of Public GitHub :
- There is 88.6k files that contains `getByEmail`
- There is 298 files that contains `getForEmail`

- There is 2.4M files that contains `getById`
- There is 7.6k files that contains `getForId`

Reply:
I totally agree with you, I personally prefer getBy instead of getFor, we can change that for me...